WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -318,6 +318,17 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// and can proceed to the self-hosted username and password view controller.
     ///
     func shouldPresentUsernamePasswordController(for siteInfo: WordPressComSiteInfo?, onCompletion: @escaping (WordPressAuthenticatorResult) -> Void) {
+        if let site = siteInfo {
+            ServiceLocator.analytics.track(event: .Login.siteInfoFetched(
```

Comment:
Can we use the injected `analytics` and also unit test this? [Like this test](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift#L438)

Reply:
Good idea! Updated in e361d74588981e5e95ed08a2d3f2cabd9530850c.